README.md

```diff
@@ -44,7 +44,7 @@ If you use Windows, you'll need to run ICE inside of [WSL](https://learn.microso
 
 1. Set required secrets in `~/.ought-ice/.env`. See [`.env.example`](https://github.com/oughtinc/ice/blob/main/.env.example) for the format.
 
-1. To learn more, go through [the Primer](https://primer.ought.org/).
+1. Run the Hello World recipe in [the Primer](https://primer.ought.org/) to see the trace rendered.
```

Comment:
I'm not sure about pushing users to jump to the first recipe without reading the intro. @stuhlmueller thoughts?

Reply:
Seems reasonable for the ICE tutorial - people probably just want to get something running and will read more as needed.

Reply:
Ok, done!

Reply:
Ok, lmk if there's anything else to do for this PR :P